src/ADAL.PCL/TokenCache.cs

```diff
@@ -392,13 +396,15 @@ internal AuthenticationResultEx LoadFromCache(CacheQueryData cacheQueryData, Cal
                     }
                     else if (!tokenExtendedLifeTimeExpired && cacheQueryData.ExtendedLifeTimeEnabled && tokenNearExpiry)
                     {
+                        cacheEvent.SetEvent(EventConstants.ExtendedExpires, "true");
                         resultEx.Result.ExtendedLifeTimeToken = true;
                         resultEx.Result.ExpiresOn = resultEx.Result.ExtendedExpiresOn;
                         PlatformPlugin.Logger.Information(callState,
                             "The extendedLifeTime is enabled and a stale AT with extendedLifeTimeEnabled is returned.");
                     }
                     else if (tokenExtendedLifeTimeExpired)
                     {
+                        cacheEvent.SetEvent(EventConstants.ExpiredAt, "true");
```

Comment:
removed

---
In reply to: [82040806](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/494#discussion_r82040806) [](ancestors = 82040806)